storybook/stories/buttons.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { PrimaryButtonV2, SecondaryButtonV2, DangerButtonV2 } from 'Components/toolbox/buttons/button';
+
+storiesOf('Button', module)
+  .add('Primary Button', () => (
+    <React.Fragment>
```

Comment:
I suggest this wrapper (and others) to have some padding so that it doesn't look like this
![Screen Shot 2019-03-26 at 11 26 33](https://user-images.githubusercontent.com/1254342/54990257-942d0300-4fba-11e9-8d0a-62e15ecd1d3f.png)

Reply:
Created StoryWrapper Component that has a padding.